server/game/gamesteps/TriggeredAbilityWindow.js

```diff
@@ -113,33 +113,58 @@ class TriggeredAbilityWindow extends BaseAbilityWindow {
         return targets.map(target => target.getShortSummary(targetsToValidate.includes(target) || this.game.isCardVisible(target, player)));
     }
 
+    getCardFromChoice(choice) {
+        return choice.context.event.card || choice.context.event.target || choice.context.event.plot;
+    }
+
     chooseCardToTrigger(player, card) {
         let choices = this.abilityChoices.filter(choice => choice.player === player && choice.card === card);
 
         if(choices.length === 0) {
             return false;
         }
 
-        let availableTargets = choices.map(choice => choice.context.event.card || choice.context.event.target).filter(card => !!card);
+        let availableTargets = choices.map(choice => this.getCardFromChoice(choice)).filter(card => !!card);
 
         if(choices.length === 1 || availableTargets.length <= 1) {
             this.chooseAbility(choices[0]);
             return true;
         }
 
+        let unclickableCards = availableTargets.filter(card => card.location === 'active plot');
+
         this.game.promptForSelect(player, {
             activePromptTitle: `Choose triggering card for ${card.name}`,
             isCardEffect: false,
             cardCondition: card => availableTargets.includes(card),
+            additionalButtons: this.getButtons(player, unclickableCards),
+            cardType: ['agenda', 'attachment', 'character', 'event', 'location', 'plot', 'title'],
             onSelect: (player, selectedCard) => {
-                let choice = choices.find(choice => choice.context.event.card === selectedCard || choice.context.event.target === selectedCard);
+                let choice = choices.find(choice => this.getCardFromChoice(choice) === selectedCard);
 
                 if(!choice || choice.player !== player) {
                     return false;
                 }
 
                 this.chooseAbility(choice);
 
+                return true;
+            },
+            onMenuCommand: (player, arg) => {
+                if(arg === 'Done') {
+                    this.pass();
+                } else {
+                    let choice = choices.find(choice => {
+                        let card = this.getCardFromChoice(choice); return card && card.uuid === arg;
+                    });
+
+                    if(!choice) {
+                        return false;
+                    }
+
+                    this.chooseAbility(choice);
+                }
+
```

Comment:
Not certain what this is doing; would remove unless absolutely vital (considering this is engine-level code).